examples/Mac/FilterShowcase/CMakeLists.txt

```diff
@@ -56,10 +56,22 @@ if(TARGET gpuimage)
       "${CMAKE_CURRENT_LIST_DIR}/../../../framework/Source/"
       "${temp_include}/GPUImage"
   )
+
   target_link_libraries(FilterShowcase PRIVATE gpuimage)
   target_include_directories(FilterShowcase PRIVATE "${temp_include}")
 else()
+
+  # FORCE LINKER TO LINK GPUImageView
+  # http://stackoverflow.com/questions/1725881/unknown-class-myclass-in-interface-builder-file-error-at-runtime
+  #
+  # Despite the "Unknown class MyClass in Interface Builder file." error printed at runtime,
+  # this issue has nothing to do with Interface Builder, but rather with the linker, which is
+  # not linking a class because no code uses it directly.
+  
+  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -all_load -ObjC")
```

Comment:
Awesome! Thanks for investigation, it's always good to know that Hunter is not responsible for build errors :) Looks like this is the only change needed to fix runtime error (at least for me). Can you please send a pull request with only this change? Also this flag only needed for one library target, so you can set it just for gpuimage instead of setting it globally (just good practice):
```cmake
target_link_libraries(gpuimage "-all_load" "-ObjC")
```
Also I've added info about error to wiki: https://github.com/ruslo/hunter/wiki/error.external.build.failed#mac-os-x